profiles/rockets/silvertip.py

```diff
@@ -41,12 +42,12 @@ def buttons(self):
     @property
     def labels(self):
         return [
-            Label(DeviceType.SILVERTIP_FLARE, "Altitude", update_altitude),
-            Label(DeviceType.SILVERTIP_FLARE, "MaxAltitude", update_max_altitude, "Max Altitude"),
-            Label(DeviceType.SILVERTIP_FLARE, "GPS", update_gps),
-            Label(DeviceType.SILVERTIP_FLARE, "State", update_state),
-            Label(DeviceType.SILVERTIP_FLARE, "Pressure", update_pressure),
-            Label(DeviceType.SILVERTIP_FLARE, "Acceleration", update_acceleration),
+            Label(DeviceType.SILVERTIP_FLARE, "Altitude", update_altitude, map_fn=receive_time_series),
```

Comment:
map_fn is used for plotting time series and plotting on the map so I added it for silvertip. map_fn = None by default though so it may cause problems when run with other rocket profiles...